packages/network-controller/tests/helpers.ts

```diff
@@ -243,7 +243,7 @@ export function buildNetworkConfiguration(
       nativeCurrency: () => 'TOKEN',
       rpcEndpoints: () => [
         defaultRpcEndpointType === RpcEndpointType.Infura
-          ? buildInfuraRpcEndpoint(InfuraNetworkType['linea-goerli'])
+          ? buildInfuraRpcEndpoint(InfuraNetworkType.sepolia)
```

Comment:
Maybe we ought to move the `TESTNET` constant to this file so that we can re-use it here?